app/src/main/java/org/tokend/template/features/wallet/adapter/BalanceChangeItemViewHolder.kt

```diff
@@ -44,11 +44,18 @@ class BalanceChangeItemViewHolder(view: View,
             actionDetailsTextView.visibility = View.VISIBLE
 
             if (counterparty != null && item.isReceived != null) {
+                val formatted = try {
```

Comment:
What if counterparty is not an account ID? Apply this format only if it's account ID, do it in `BalanceChangeListItem` constructor